### PR TITLE
YAML meta data updates

### PR DESCRIPTION
**Pull request by crgrieve**
Updates to meta data as per YAML docs

### CODE REVIEW

File: Development-Guidelines/Coding-Standards/index.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 versionFrom: 7.0.0
+meta.Title: "Coding Standards"
+meta.Description: "A guide to coding standards and naming conventios in Umbraco core."
```
Comment (sofietoft) [inline Development-Guidelines/Coding-Standards/index.md]:
```suggestion
meta.Description: "A guide to coding standards and naming conventions in Umbraco core."
```

File: Development-Guidelines/index.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 versionFrom: 7.0.0
+meta.Title: "Development Guidelines"
+meta.Description: "A guide to developing in umbraco core codebase"
```
Comment (sofietoft) [inline Development-Guidelines/index.md]:
```suggestion
meta.Description: "A guide to developing in the Umbraco core codebase"
```

File: Extending/Content-Apps/index.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 versionFrom: 8.0.0
+meta.Title: "Content Apps"
+meta.Description: "A gudie to Umbraco Content Apps in backoffice"
```
Comment (sofietoft) [inline Extending/Content-Apps/index.md]:
```suggestion
meta.Description: "A guide to Umbraco Content Apps in the backoffice"
```

**Comment by crgrieve** [comment]
Oops, typos!

I have merged those suggestions :)

**Comment by sofietoft** [comment]
Thanks for seeing to the typos 😄 

Also looks like the linter passes this time 💪 
I'll merge it.

Thanks again @crgrieve !

**Pull request merged**